container-search/src/main/java/com/yahoo/search/Query.java

```diff
@@ -293,20 +303,24 @@ public Query(String query, CompiledQueryProfile queryProfile) {
      * @param queryProfile the query profile to use for this query, or null if none.
      */
     public Query(HttpRequest request, CompiledQueryProfile queryProfile) {
-        super(new QueryPropertyAliases(propertyAliases));
-        this.httpRequest = request;
-        init(request.propertyMap(), queryProfile);
+        this(request, request.propertyMap(), queryProfile);
     }
 
     /**
-     * Creates a query from a request
+     * Creates a query from a request containing a JSON-query.
      *
```

Comment:
No need to mention JSON here - it is just creating a request from a map.

Reply:
Removed 👍